### PR TITLE
Add an "in" comparison operator

### PR DESCRIPTION
**Pull request by RobMullen**
During recent plugin development I had need to terminate my repeat on any of a set of results.  Thought it might be useful to others.

### CODE REVIEW

File: src/test/ts_test_config.erl [hunk 1]
```diff
@@ -223,7 +223,11 @@ cport_list_node_test() ->
 
 
 ifalias_test() ->
-    Res=ts_ip_scan:get_intf_aliases("lo"),
+    Interface = case os:type() of
+		    {unix,darwin} -> "lo0";
+		    _ -> "lo"
+		end,
+    Res=ts_ip_scan:get_intf_aliases(Interface),
```
Comment (tisba) [inline src/test/ts_test_config.erl]: Not sure how this change relates to your feature...

File: src/tsung/ts_ip_scan.erl [hunk 2]
```diff
@@ -157,17 +157,27 @@ code_change(_OldVsn, State, _Extra) ->
...
+	_ ->
+	    case file:read_file_info("/sbin/ip") of
+		{ok,_} ->
+		    Res=os:cmd("LC_ALL=C /sbin/ip -o -f inet addr show dev "++Interface),
+		    get_ip_aliases(string:tokens(Res,"\n"), []);
+		{error,Reason} ->
+		    ?LOGF("ip command not found (~p), using ifconfig instead~n",[Reason],?NOTICE),
+		    Res=os:cmd("LC_ALL=C /sbin/ifconfig "),
+		    get_intf_aliases(string:tokens(Res,"\n"), Interface,[],[])
+	    end
```
Comment (tisba) [inline src/tsung/ts_ip_scan.erl]: Again, how this relates to your feature? Maybe committed by accident?

**Comment by tisba** [comment]
This is a nice feature, thanks!

**Comment by RobMullen** [comment]
Thank you for checking this Sebastian.  I'm sorry for the sloppy PR.  My poor job pulling this from a dev branch.  I'll just close this one and file two new separate ones ( one for the "in" operator and another to address the /sbin/ip dependency ).  Best regards

**Pull request closed without merging**